x/wasm/client/cli/gov_tx.go

```diff
@@ -80,6 +86,7 @@ func ProposalStoreCodeCmd() *cobra.Command {
 	cmd.Flags().String(flagInstantiateByEverybody, "", "Everybody can instantiate a contract from the code, optional")
 	cmd.Flags().String(flagInstantiateNobody, "", "Nobody except the governance process can instantiate a contract from the code, optional")
 	cmd.Flags().String(flagInstantiateByAddress, "", "Only this address can instantiate a contract instance from the code, optional")
+	cmd.Flags().Bool(flagPinCode, false, "Pin code on upload, optional")
```

Comment:
I agree with this PR in general.
I would change the default to be pinning, and require explicitly unpinning.

You can safely pin a few hundred contracts, and I assume most ones on permissioned chains would be high-value.

Reply:
Best way may be to rename to "unpinContract" or so, so default (no protobuf field, no cli flag) would be the same as the current behavior.

Reply:
@ethanfrey thanks for the feedback, so just to get things clear before updating the PR.

You mean only change the flag ?
```
cmd.Flags().Bool(flagUnpinContract, false, "Unpin contract as part of the proposal, optional")
``` 

But send `PinCode: true` as default unless the flag is enabled ?

Reply:
The issue is with in flight gov proposals. The default of the new (proto) field would be false and,changes the behaviour. Or we need to provide a store migration to set the field too true. 
When you rename the field to something like `unpinned` then the unset field would default to false and the code can run as before with the same outcome.

Reply:
What Alex said, change the field and change the flag.
Default behaviour remains the same, unless they explicitly add it

Reply:
Changed